### ISSUE TITLE
The dimensions of the training set and the test set do not match

### ISSUE THREAD

**Issue by Joy2Boy** [openclimatefix/skillful_nowcasting#83]
I want to know if the dimension of the training set is 256x256 and the test set is 512x512, can such a model perform inference? Also, can the author provide the inference code for reference, because the inference code on deep mind seems to be read remotely through tfhub.
